frontend/src/components/pages/CampCreation/StepperNavigation/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Box, Center } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCampground,
+  faCalendarDays,
+  faCircleInfo,
+} from "@fortawesome/free-solid-svg-icons";
+import StepperTab from "./StepperTab";
+
+export enum CampCreationPages {
+  CampCreationDetailsPage,
+  ScheduleSessionsPage,
+  RegistrationFormPage,
+}
+
+type StepperProps = {
+  currentPage: CampCreationPages;
+  setCurrentPage: React.Dispatch<React.SetStateAction<CampCreationPages>>;
```

Comment:
What's the difference between this and `(campCreationPage: CampCreationPage) => void`?